packages/datadog-plugin-http/src/client.js

```diff
@@ -169,13 +169,16 @@ function normalizeClientConfig (config) {
   const propagationFilter = getFilter({ blocklist: config.propagationBlocklist })
   const headers = getHeaders(config)
   const hooks = getHooks(config)
+  const enablePropagationWithAmazonHeaders = config.enablePropagationWithAmazonHeaders ||
+    process.env.DD_TRACE_FORCE_AWS_PROPAGATION === 'true'
```

Comment:
This environment variable should affect the above configuration option instead of being hardcoded in this file.